modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i

```diff
@@ -211,4 +211,5 @@
   exodus = true
   print_linear_residuals = true
   print_perf_log = true
+  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
```

Comment:
The variables don't necessarily need to be hidden; penetration and contact pressure are also automatic and do not get hidden. I just hid these to make the tests pass without changes to the gold files.

I have no preference on hiding them by default, assuming that is possible to do in a clean way.